src/index.js

```diff
@@ -1,50 +1,123 @@
-import checkItem from './status.js';
-import './style.css';
-
-const tasks = [
-  {
-    description: 'Finish my first project',
-    completed: false,
-    id: 1,
-  },
-  {
-    description: 'Finish my second project',
-    completed: false,
-    id: 2,
-  },
-  {
-    description: 'Finish my third project',
-    completed: false,
-    id: 3,
-  },
-  {
-    description: 'Finish my third project',
-    completed: false,
-    id: 4,
-  },
-];
-
-if (!localStorage.getItem('todo')) {
-  localStorage.setItem('todo', JSON.stringify(tasks));
-}
-
-const todos = JSON.parse(localStorage.getItem('todo') || '[]');
-
-const todoList = document.querySelector('.todoList');
-
-todos.forEach(({ description, id, completed }) => {
-  const div = document.createElement('div');
-  div.classList.add('d-height');
-  div.classList.add('todo-cont');
-  div.innerHTML = `<li><input type="checkbox" ${completed && 'checked'} class="check-box"   id="check-${id}">${description}</li>
-        <div class="delete-icon"></div>`;
-  todoList.appendChild(div);
-});
-
-for (let i = 0; i < todos.length; i += 1) {
-  const inCheck = document.querySelector(`#check-${todos[i].id}`);
-  inCheck.addEventListener('change', (e) => {
-    checkItem(i, e, todos);
+window.onload = () => {
+  const todos = JSON.parse(localStorage.getItem('todo') || '[]');
+  const todoList = document.querySelector('.todoList');
+
+  todos.forEach(({ description, id, completed }) => {
+    todoList.innerHTML += `
+      <li class="listElements">
+        <div class="d-flex center-items list-group-1">
+          <input type="checkbox" ${completed && 'checked'} class="check-box" id="check-${id}">
+          <p class="task-description ${completed ? 'text-line' : ''}">${description}</p>
+          <input type="input" class="edit-input no-outline">
+        </div>
+        <div>
+          <i class="fas fa-trash-alt trash"></i>
+          <i class="fas fa-ellipsis-v"></i>
+        </div>
+      </li>
+      <hr>`;
+  });
+
+  const addTask = (entry) => {
+    todos.push(entry);
+    localStorage.setItem('todo', JSON.stringify(todos));
+    window.location.reload();
+  };
+
+  const deleteTask = (index) => {
+    const currentTasks = todos.filter((todo, todoIndex) => todoIndex !== index);
+    localStorage.setItem('todo', JSON.stringify(currentTasks));
+    window.location.reload();
+  };
+
```

Comment:
- [ ] It seems that for now your delete feature is not yet working, because when I try deleting a task it does not seems to be deleted. Kindly let's fix this issue 😉